SpatialGDK/Documentation/toc.md

```diff
@@ -20,31 +20,34 @@
     - [Unreal features support]({{urlRoot}}/unreal-features-support)
     - [Known issues]({{urlRoot}}/known-issues)
 - <h3>Concepts and terminology</h3>
-    - SpatialOS concepts
-        - [Summary]({{urlRoot}}/content/spatialos-concepts/concepts)
-        - [Schema]({{urlRoot}}/content/spatialos-concepts/schema)
-        - [Snapshots]({{urlRoot}}/content/spatialos-concepts/generating-a-snapshot) 
+    - [SpatialOS concepts]({{urlRoot}}/content/spatialos-concepts/concepts)
     - [Glossary]({{urlRoot}}/content/glossary)
     - [Toolbar]({{urlRoot}}/content/toolbars)
 - <h3>Reference</h3>
-    - [Spatial Type]({{urlRoot}}/content/spatial-type)
-    - [Dynamic Typebindings]({{urlRoot}}/content/dynamic-typebindings)
-    - [Unreal networking authority]({{urlRoot}}/content/authority)
-    - [Singleton Actors]({{urlRoot}}/content/singleton-actors)
-    - [Actor handover between server-workers]({{urlRoot}}/content/handover-between-server-workers)
-    - [Map travel]({{urlRoot}}/content/map-travel)
-    - [Cross-server RPCs]({{urlRoot}}/content/cross-server-rpcs)
-    - [Helper scripts]({{urlRoot}}/content/helper-scripts)
-    - [Directory structure]({{urlRoot}}/content/directory-structure)
+    - Actors and entities
+        - [Schema]({{urlRoot}}/content/spatialos-concepts/schema)
+        - [Spatial Type]({{urlRoot}}/content/spatial-type)
+        - [Dynamic Typebindings]({{urlRoot}}/content/dynamic-typebindings)
+        - [Singleton Actors]({{urlRoot}}/content/singleton-actors)
```

Comment:
FYI - while Singleton Actors are really more about authority, sorta, it seemed counter-intuitive to put this page into "authority". :-)